client/src/Forside/Forside.tsx

```diff
@@ -64,7 +64,8 @@ export const Forside: FunctionComponent<ForsideProps> = ({
   }, [orgnr, miljø]);
 
   const sykefraværshistorikk = useSykefraværshistorikk();
-  const kvartalsvisSykefraværshistorikkData = sykefraværshistorikk.status === RestStatus.Suksess
+  const kvartalsvisSykefraværshistorikkData =
+    sykefraværshistorikk.status === RestStatus.Suksess
```

Comment:
Hadde foretrukket om dette ble splittet opp til to statements siden jeg synes det ville vært litt ryddigere.

Reply:
Jeg synes også det hadde vært bedre med en boolean som f.eks `erSykefraværsshistorikkLastetNed`, men det medfører et problemet ved å gjøre det slik. 
Hvis vi tar ut `sykefraværshistorikk.status === RestStatus.Suksess` til f.eks `erSykefraværsshistorikkLastetNed`, da vil TypeScript ikke forstå at  `sykefraværshistorikk.data` er tilgjengelig (pga type som er `RestRessurs<T>` hvor `T` bare er tilgjengelig ved verdi `Suksess<T>`. 
Må dessverre la det være som det er foreløpig.

Reply:
Mulig dette kan fikses med type predicates, men det gjør kanskje det litt mer rotete.

https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Reply:
Prøvde med Type Predicate. Synes det er bedre med det, takk for tips 👍